src/store/modules/product/mutations.ts

```diff
@@ -8,7 +8,12 @@ const mutations: MutationTree <ProductState> = {
       payload.products.forEach((product: any) => {
         state.cached[product.productId] = product
       });
+      state.products.total = payload.totalProductsCount
```

Comment:
Why we need this change?

Reply:
products: {
list: any,
total: number
}
Sir, Because I Updated the product state.